static/app/views/settings/project/server-side-sampling/serverSideSampling.tsx

```diff
@@ -503,11 +503,8 @@ export function ServerSideSampling({project}: Props) {
         />
         <TextBlock>
           {tct(
-            'Improve the accuracy of your [performanceMetrics: performance metrics] and [targetTransactions: target those transactions] which are most valuable for your organization. Server-side rules are applied immediately, without having to re-deploy your app.',
+            'Improve the accuracy of your performance metrics and [targetTransactions: target those transactions] which are most valuable for your organization. Server-side rules are applied immediately, without having to re-deploy your app.',
             {
-              performanceMetrics: (
-                <ExternalLink href="https://docs.sentry.io/product/performance/metrics/#metrics-and-sampling" />
```

Comment:
Raised a question about this in the PR https://github.com/getsentry/sentry-docs/pull/5688#discussion_r1007859711 and maybe this won't be removed